ssmlib/backends/crypt.py

```diff
@@ -197,6 +197,10 @@ def __init__(self, *args, **kwargs):
             dm['real_dev'] = misc.get_real_device(devname)
             if dm['real_dev'] in self.mounts:
                 dm['mount'] = self.mounts[dm['real_dev']]['mp']
+            else:
+                for swap in misc.get_swaps():
+                    if swap[0] == dm['real_dev']:
+                        dm['mount'] = "SWAP"
```

Comment:
Please move the get_swaps() to the init method of the base class. That way you don't have to call it for every device you're processing. This applies to all the backends you're making this change for

Reply:
I would also like to see some added tests for this.

Reply:
crypt bash tests and multipath bash tests are broken

Reply:
looking further, it seems the crypt bash tests expect luks1 and my system is luks2; will look at this more later

Reply:
crypt bash testing fixed and swap test added